### PR TITLE
Implement base GUI of displaying tasks

### PR DESCRIPTION
**Pull request by dannytayjy**
- Incomplete, still in the process of fixing

### CODE REVIEW

File: src/main/java/manageezpz/ui/TaskCard.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        if (!(other instanceof TaskCard)) {
+            return false;
+        }
+
+        // state check
+        TaskCard card = (TaskCard) other;
+        return id.getText().equals(card.id.getText())
+                && task.equals(card.task);
+    }
+}
```
Comment (ChanWeiJie) [inline src/main/java/manageezpz/ui/TaskCard.java:152]: Good job on creating a seprate java class for the Tasks!

File: src/main/resources/view/DarkTheme.css [hunk 2]
```diff
@@ -132,6 +132,30 @@
...
+    -fx-text-fill: #eaeaea;
+}
+
+.cell_priority_label {
+    -fx-background-color: #4285f4;
+    -fx-background-radius: 5;
+    -fx-font-family: "Segoe UI";
+    -fx-font-size: 13px;
+    -fx-text-fill: #eaeaea;
+}
```
Comment (ChanWeiJie) [inline src/main/resources/view/DarkTheme.css:157]: Great work on setting the different colours ! :D

File: src/main/java/manageezpz/logic/Logic.java [hunk 3]
```diff
@@ -33,6 +34,9 @@
     /** Returns an unmodifiable view of the filtered list of persons */
     ObservableList<Person> getFilteredPersonList();
 
+    /** Returns an unmodifiable view of the filtered list of tasks */
+    ObservableList<Task> getFilteredTaskList();
+
```
Comment (alfredkohhh) [inline src/main/java/manageezpz/logic/Logic.java:39]: Hey thanks for adding this logic in !

File: src/main/java/manageezpz/ui/MainWindow.java [hunk 4]
```diff
@@ -110,8 +114,11 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
      * Fills up all the placeholders of this window.
      */
     void fillInnerParts() {
-        personListPanel = new PersonListPanel(logic.getFilteredPersonList());
-        personListPanelPlaceholder.getChildren().add(personListPanel.getRoot());
+        /*personListPanel = new PersonListPanel(logic.getFilteredPersonList());
+        personListPanelPlaceholder.getChildren().add(personListPanel.getRoot());*/
+
+        taskListPanel = new TaskListPanel(logic.getFilteredTaskList());
+        taskListPanelPlaceholder.getChildren().add(taskListPanel.getRoot());
```
Comment (alfredkohhh) [inline src/main/java/manageezpz/ui/MainWindow.java:121]: Hey, as discussed, good job @dannytayjy for helping to deal with the refresh state of tasklist

**Review by alfredkohhh — APPROVED** [review]
Hey @dannytayjy , thanks for single handedly doing the GUI, much appreciated !!

**Pull request merged**